mastering-plone/events.md

```diff
@@ -28,7 +28,7 @@ git checkout talklist
 Code for the end of this chapter:
 
 ```
-# frontent
+# frontend
```

Comment:
`#` is now a heading in `.md` when not in a `code-block`. I think we need to put this into a codefence with triple backticks and set the language to `shell`.